apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/add-edit-notification-dialog.component.html

```diff
@@ -149,11 +149,15 @@ <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TITLE_EDIT' | transl
                             *ngIf="format === 'plain'"
                             formControlName="{{lang}}-plain-subject"
                             (focus)="isTextFocused = false"
+                            [value]="this.applicationMail.message[lang].subject"
+                            (input)="this.applicationMail.message[lang].subject = inputFormGroup.controls[lang + '-plain-subject'].value"
```

Comment:
I believe this should not be needed anymore. Also the way I understand it, is that the form control serves exactly this purpose of input-value connection (with some additional features).

Reply:
Yes. Once I removed the duplicities, the problem no longer occurs.